### ISSUE TITLE
Improve onboarding experience when user is connecting resources

### ISSUE THREAD

**Issue by I-Dont-Remember** [windmill-labs/windmill#1200]
*tl;dr connecting resources is a friction point for new users before they see the value of Windmill - try to catch them by making it a smoother process*

---
As a first time user trying out Windmill, I played around in the demo for a little while, but very quickly wanted to get my hands dirty with some of my own databases/tools. 

It seems like a potential point for onboarding users to drop off if there's too much friction before they can try out Windmill with their own data; since connecting to databases/AWS/etc. isn't a part that shows off the value prop of Windmill.

One small improvement towards that end, though I don't think it should be considered finalized, is helping people that want to connect to databases behind firewalls/security groups from the hosted Windmill app. An example of this is Retool providing specific instructions on the IPs that need to be white-listed before a DB connection will successfully go through.

![image](https://user-images.githubusercontent.com/8552248/218589578-58838b8e-7ddb-40d9-9e53-487431d3d7b7.png)

**Comment by rubenfiszel** [comment]
Thank you for the effort you put into this issue.

The example you've shown for mysql is a great one. There are 2 subtasks for us:
1) have a set of static ips that we would show in the frontend (and in the documentation if they are truly static)
2) show them at time of adding a resource for some types 

For 2),  the set can be hardcoded in the frontend and one can already retrieve the workers actual ips.
For 1), a little bit of work need to be done as we are using eks but it seems that this might do the trick: https://blog.damavis.com/en/adding-static-outbound-ips-in-amazon-eks/

**Comment by rubenfiszel** [comment]
Adding to this issue, adding a connection tester for postgresql and mysql

**Comment by rubenfiszel** [comment]
![Screenshot 2023-02-15 at 19-51-37 Resources Windmill](https://user-images.githubusercontent.com/275584/219124902-246d0807-750f-4ecd-bf75-89b104127864.png)

In latest commit, the first feature request is done. IPs to whitelist are shown for postgresql and mysql

**Comment by rubenfiszel** [comment]
![Screenshot 2023-02-15 at 21-20-06 Resources Windmill](https://user-images.githubusercontent.com/275584/219147225-126756f1-0199-4ac8-8594-edf769ff39ec.png)
second item also done
